SPEC_MAINTAINER_PROTOCOL.md

```diff
@@ -0,0 +1,80 @@
+# Spec Maintainer Protocol
+
+> We have a protocol to maintain specs of protocols! How meta, but very useful.
+
+## Motivation
+
+The protocol specifications should be treated as the ultimate source of truth for how the protocol works, how to implemented and what users and builders can expect on how it behaves. However, writting specs for a protocol is not novelty and there has been multiple decades of research on how to do this well, from having a reference implementation, standards bodies and/or formal verification to ensure both code and specs behave the same. It is a hard problem and ultimately one that questions how programming is done as a whole.
```

Comment:
```suggestion
The protocol specifications should be treated as the ultimate source of truth for how the protocol works, how to be implemented and what users and builders can expect on how it behaves. However, writing specs for a protocol is not novelty and there has been multiple decades of research on how to do this well, from having a reference implementation, standards bodies and/or formal verification to ensure both code and specs behave the same. It is a hard problem and ultimately one that questions how programming is done as a whole.
```

Reply:
The diff is hard to see what has been changed, so to make clear:
s/how to implemented/how to be implemented
s/writting/writing